core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -445,9 +445,14 @@ public enum Property {
   MANAGER_SPLIT_WORKER_THREADS("manager.split.inspection.threadpool.size", "8", PropertyType.COUNT,
       "The number of threads used to inspect tablets files to find split points.", "4.0.0"),
 
-  MANAGER_COMPACTION_SERVICE_PRIORITY_QUEUE_SIZE("manager.compaction.major.service.queue.size",
-      "10000", PropertyType.COUNT,
-      "The max size of each resource groups compaction job priority queue.", "4.0"),
+  MANAGER_COMPACTION_SERVICE_PRIORITY_QUEUE_INITIAL_SIZE(
+      "manager.compaction.major.service.queue.initial.size", "10000", PropertyType.COUNT,
+      "The initial size of each resource groups compaction job priority queue.", "4.0.0"),
+  MANAGER_COMPACTION_SERVICE_PRIORITY_QUEUE_SIZE_FACTOR(
+      "manager.compaction.major.service.queue.size.factor", "1.5", PropertyType.FRACTION,
```

Comment:
A higher default give more opportunity for the TGW to buffer jobs in the case where the jobs times are short.

```suggestion
      "manager.compaction.major.service.queue.size.factor", "3", PropertyType.FRACTION,
```

Reply:
Changed to 3.0 in 18d2243